### PR TITLE
alignment with site grade logic from extensions team

### PR DESCRIPTION
**Pull request by brindy**
Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/458611166160477/f
CC:

**Description**:

Aligns the Site Rating with the extensions team and tries to make main logic section look similar so it's easier to track future changes.

Additionally, fixes some feedback raised with initial screen (which are mostly related to this work):

* Privacy Grade from D to B doesn't show https://app.asana.com/0/413877547933097/312386120368398/f

* Show privacy grade while loading: https://app.asana.com/0/413877547933097/312386120368396/f

* Slow protection toggle: https://app.asana.com/0/413877547933097/312386120368391/f

* Unknown privacy practices should be rated B max: https://app.asana.com/0/413877547933097/312386120368386/f

* Minor change to improve UX of dismissing privacy protection while under consideration by Mike: https://app.asana.com/0/413877547933097/312386120368389/f

Also updated the assets with the vector PDFs.

Note: have added hooks to handle non-secure content as we currently base https purely on the site's URL, but that's too big a change for here, so will create a new Asana to integrate that.

**Steps to test this PR**:

Test 1:
1. Visit Evans Cycles and launch the Privacy Protection
1. Note the upgrade message
1. Turn off privacy protection
1. Note that the toggle is instant
1. Note the grade updates
1. Turn on privacy protection
1. Note the upgrade message

Test 2:
1. Visit news.ycombinator.com
1. Privacy Protection 
1. Note the B grade

Test 3:
1. Visit a site and launch the Privacy Protection
1. Tap the omni bar text field
1. The privacy protection is dismissed and the text field takes focus

###### Reviewer Checklist:
- [x] **Ensure the PR solves the problem**
- [x] **Review every line of code**
- [x] **Ensure the PR does no harm by testing the changes thoroughly**
- [x] **Get help if you're uncomfortable with any of the above!**
- [x] Determine if there are any quick wins that improve the implementation

###### PR DRI Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications (Database connections, Grafana stats, CPU)

### CODE REVIEW

File: Core/TermsOfService.swift [hunk 1]
```diff
@@ -23,9 +23,24 @@ import Foundation
 
 public struct TermsOfService {
 
-    public let classification: Classification?
-    public let score: Int
-    
+    private static let classificationScores = [
```
Comment (subsymbolic) [inline Core/TermsOfService.swift]: Optional, reformat to  more common enum `.`  reference e.g:
```
private static let classificationScores: [Classification: Int] = [
        .a: -1,
        .b: 0,
        .c: 0,
        .d: 1,
        .e: 2
    ]
```

Comment (brindy) [reply Core/TermsOfService.swift]: Yeah I agree, that's nicer.

File: Core/SiteRating.swift [hunk 2]
```diff
@@ -27,33 +27,25 @@ public class SiteRating {
...
-        if let network = MajorTrackerNetwork.network(forDomain: domain) {
-            return network
-        }
-        
-        if let associatedDomain = disconnectMeTrackers.filter( { domain.hasSuffix($0.key) } ).first?.value.parentDomain {
-            return MajorTrackerNetwork.network(forDomain: associatedDomain)
-        }
-            
-        return nil
+        return url.isHttps() && insecureContentCount == 0
```
[the remark this answers is not in the document]
Comment (brindy) [reply Core/SiteRating.swift]: Yep, totally.  Thanks for that.

**Comment by brindy** [comment]
@subsymbolic 

Updated with your feedback.

I've added a "mixed encryption" state, you can test it by going to https://www.falkirkrpg.org.uk/mixed.html

So use DDG to test for full https state, example.com for unencrypted state and the above for mixed.

**Comment by subsymbolic** [comment]
Looks good!

**Pull request merged**